index.html

```diff
@@ -50,14 +50,19 @@ <h1>CODEVID-19</h1>
         make a difference in your community and the fight against COVID-19 by
         joining CODEVID-19, the world’s first pandemic hackathon!
       </p>
+      <p>
```

Comment:
I actually wanted to keep this out of the first fold permanently but I have to jet...